docs/docusaurus/sidebars.json

```diff
@@ -260,26 +260,9 @@
       }
     ],
     "Contribute": [
-      {
-        "type": "subcategory",
-        "label": "Contribute",
-        "ids": [
-          "contributing/contribute_onboarding",
-          "contributing/contribute_workflow",
-          "contributing/contribute_vscode",
-          "contributing/contribute_conventions",
-          "contributing/contribute_id_mappings",
-          "contributing/contribute_ci_checks"
```

Comment:
Shouldn't these files be removed from the readmes folder if we don't want them in upcoming versions?

Reply:
> Shouldn't these files be removed from the readmes folder if we don't want them in upcoming versions?

I wanted to verify that step socially before doing something destructive. You replying here gets that going. What I'll do now is remove those and re-push. If you or @Neudrino raises a flag I can reinstate.

Reply:
See https://github.com/magma/magma/pull/14235/commits/ff64264754218f88ccc407fd274688f97dc24684